src/ompl/geometric/planners/quotientspace/algorithms/MultiQuotient.ipp

```diff
@@ -0,0 +1,322 @@
+/*********************************************************************
```

Comment:
OMPL uses `*.h` files for implementation of templated class methods (e.g., the nearest neighbor data structures). Can you change this file to `MultiQuotient_impl.h` rather than `.ipp`?